src/components/Button.tsx

```diff
@@ -96,8 +98,8 @@ export function ButtonRedirect({
       property={`fc:frame:button:${index}:action`}
       content="post_redirect"
     />,
-    location && (
-      <meta property={`fc:frame:button:${index}:target`} content={location} />
+    target && (
+      <meta property={`fc:frame:button:${index}:target`} content={target} />
```

Comment:
`location` is where to redirect, `target` is where to post the action to get the `location`

Reply:
still need to make sure `target` works correctly